deepcell/applications/application_test.py

```diff
@@ -29,9 +29,10 @@
 from __future__ import division
 from __future__ import print_function
 
+from itertools import product
 import numpy as np
+from unittest.mock import Mock
```

Comment:
```suggestion
from itertools import product
from unittest.mock import Mock

import numpy as np
```

Reply:
Just so that imports with the same structure are next to each other? Or because we're importing numpy with an abbreviation?

Reply:
It's a convention to group imports by future, built-in, 3rd party, custom/local module imports.